docs/use-cases/deploy-ml.md

```diff
@@ -1,77 +1,114 @@
+
+You can do all of this using the [Viam app](https://app.viam.com) user interface.
+You will not need to write any code.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Create a dataset and label data](#create-a-dataset-and-label-data)
+2. [Train and test a machine learning (ML) model](#train-and-test-a-machine-learning-ml-model)
+
+{{< /alert >}}
+
+## Create a dataset and label data
 
 {{< table >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
-**1. Collect**
+{{<imgproc src="/services/icons/data-capture.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Collect data">}}
+**1. Collect images**
+
+Start by collecting images from your cameras and syncing it to the [Viam app](https://app.viam.com).
+See [Collect image data and sync it to the cloud](/use-cases/image-data/#collect-image-data-and-sync-it-to-the-cloud) for instructions.
+
+<br>
+{{% alert title="Tip" color="tip" %}}
+To keep your data organized, configure a tag in your data management service config panel.
+This tag will be applied to all data synced from that machine.
+If you apply the same tag to all data gathered from all machines that you want to use in your dataset, you can filter by that tag in the Viam app **DATA** tab to make the next steps easier.
+
+This is not required, since you can use other filters like time or machine ID in the **DATA** tab to isolate your data.
+{{% /alert %}}
+
+{{< /tablestep >}}
+{{< tablestep >}}
+{{<imgproc src="/services/ml/collect.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Label data">}}
+**2. Label your images [_(i)_](/services/data/dataset/)**
 
-Start by collecting images from your cameras with the [data management service](/services/data/).
-You can [view the data](/services/data/view/) on the **Data tab**.
+Once you have enough images of the objects you'd like to classify, use the interface on the **DATA** tab to label your data.
+If you want to train an image classifier, use image tags.
+For an object detector, use bounding boxes.
 
 {{< /tablestep >}}
 {{< tablestep >}}
-{{<imgproc src="/services/ml/label.svg" class="fill alignleft" style="max-width: 300px" declaredimensions=true alt="Label data">}}
-**2. Create a dataset and label**
+{{<imgproc src="/services/ml/label.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="Label data">}}
+**2. Create a dataset [_(i)_](/services/data/dataset/)**
 
-Once you have enough images of the objects you'd like to classify, [label your data and create a dataset](/services/data/dataset/) in preparation for training classification models.
+Use the interface on the **DATA** tab (or the [`viam data dataset add` command](/cli/#data)) to add all images you want to train the model on to a dataset.
 
 {{< /tablestep >}}
+{{< /table >}}
```

Comment:
```suggestion
{{< /table >}}

{{% alert title="Tip" color="tip" %}}
To keep your data organized, you can configure a tag in your data management service config panel.
This tag will be applied to all data synced from that machine in the future.
If you apply the same tag to all data gathered from all machines that you want to use in your dataset, you can filter by that tag in the Viam app **DATA** tab, or when querying data.

This is not required, since you can use other filters like time or machine ID in the **DATA** tab to isolate your data.
{{% /alert %}}
```